libs/interpreter-lib/src/interpreter.spec.ts

```diff
@@ -26,4 +27,141 @@ describe('Interpreter', () => {
       expect(exitCode).toEqual(ExitCode.SUCCESS);
     });
   });
+
+  describe('hooks', () => {
```

Comment:
Can you add a test to check if the input/output is called with the correct value?